src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -168,9 +168,11 @@ public bool TryWriteProperties(PropertiesWriter writer, out IEnumerable<ProjectD
         private ICollection<FileInfo> PutFilesToRightModuleOrRoot(IEnumerable<ProjectData> projects, DirectoryInfo baseDirectory)
         {
             var fileWithProjects = projects
-                .SelectMany(p => p.ReferencedFiles.Select(f => new { Project = p, File = f }))
+                .SelectMany(p => p.ReferencedFiles.Where(f => !f.Extension.Equals(".exe") && !f.Extension.Equals(".dll"))
```

Comment:
It might be an overkill, however I suggest to do the  `.exe` and `.dll` comparisons case insensitive.
Also add a test case that verifies this behavior.